doc/command_ref.rst

```diff
@@ -136,7 +136,7 @@ Options
     Disable all plugins.
 
 ``-q, --quiet``
-    quiet operation
+    In combination with non-interactive command it shows just relevant content. It suppress messages notifying about current state or actions of DNF.
```

Comment:
It's very likely that I'm wrong but I believe that
- there should be a comma after the "command": `In combination with a non-interactive command, it`
- there should be an article before "non-interactive command": `with a non-interactive command`
- there should be an article before "relevant content": `just the relevant content`
- it should be "suppresses" instead of just `suppress`
- people may wonder what happens if it is combined with an interactive command... I think that it prints just the relevant information as well, doesn't it?